newIDE/app/src/EventsSheet/EventsTree/style.css

```diff
@@ -9,19 +9,20 @@
 /**
  * Draggable handle on the left of an event
  */
-.gd-events-sheet .rst__moveHandle {
+.gd-events-sheet .move-handle {
     width: 10px;
-    border: none;
-    box-shadow: none;
-    border-radius: 0;
+    margin: 0.5px;
```

Comment:
If the question is "why I added it", it is because react-sortable-tree had a default border in their .movehandle class (that we do not use anymore). If it is "why this specific value", it is the one that felt the closest to what we have right now :)

Reply:
Ok thanks 👍